presto-raptor/src/test/java/com/facebook/presto/raptor/metadata/TestRaptorMetadata.java

```diff
@@ -141,6 +141,86 @@ public void testRenameColumn()
     }
 
     @Test
+    public void testDropColumn()
+            throws Exception
+    {
+        assertNull(metadata.getTableHandle(SESSION, DEFAULT_TEST_ORDERS));
+        metadata.createTable(SESSION, buildTable(ImmutableMap.of(), tableMetadataBuilder(DEFAULT_TEST_ORDERS)
+                .column("orderkey", BIGINT)
+                .column("price", BIGINT)));
+        ConnectorTableHandle tableHandle = metadata.getTableHandle(SESSION, DEFAULT_TEST_ORDERS);
+        assertInstanceOf(tableHandle, RaptorTableHandle.class);
+
+        RaptorTableHandle raptorTableHandle = (RaptorTableHandle) tableHandle;
+
+        ColumnHandle lastColumn = metadata.getColumnHandles(SESSION, tableHandle).get("orderkey");
+        metadata.dropColumn(SESSION, raptorTableHandle, lastColumn);
+        assertNull(metadata.getColumnHandles(SESSION, tableHandle).get("orderkey"));
+    }
+
+    @Test
+    public void testDropDisallowedColumn()
+            throws Exception
+    {
+        assertNull(metadata.getTableHandle(SESSION, DEFAULT_TEST_ORDERS));
+        Map<String, Object> properties = ImmutableMap.of(
+                BUCKET_COUNT_PROPERTY, 16,
+                BUCKETED_ON_PROPERTY, ImmutableList.of("orderkey"),
+                ORDERING_PROPERTY, ImmutableList.of("totalprice"),
+                TEMPORAL_COLUMN_PROPERTY, "orderdate");
+        ConnectorTableMetadata ordersTable = buildTable(properties, tableMetadataBuilder(DEFAULT_TEST_ORDERS)
+                .column("orderkey", BIGINT)
+                .column("totalprice", DOUBLE)
+                .column("orderdate", DATE)
+                .column("highestid", BIGINT));
+        metadata.createTable(SESSION, ordersTable);
+
+        ConnectorTableHandle ordersTableHandle = metadata.getTableHandle(SESSION, DEFAULT_TEST_ORDERS);
+        assertInstanceOf(ordersTableHandle, RaptorTableHandle.class);
+        RaptorTableHandle ordersRaptorTableHandle = (RaptorTableHandle) ordersTableHandle;
+        assertEquals(ordersRaptorTableHandle.getTableId(), 1);
+
+        assertInstanceOf(ordersRaptorTableHandle, RaptorTableHandle.class);
+
+        // disallow dropping bucket, sort, temporal and highest-id columns
+        ColumnHandle bucketColumn = metadata.getColumnHandles(SESSION, ordersRaptorTableHandle).get("orderkey");
+        try {
```

Comment:
I think we could add a helper for this:
```java
assertThrows("Cannot drop bucket columns in a table", () ->
        metadata.dropColumn(SESSION, ordersTableHandle, bucketColumn));
```
TestNG's `Assert` class has a similar helper but it compares a class, not a message